perses/rjmc/geometry.py

```diff
@@ -733,6 +732,9 @@ def _corrected_reduced_potential(self, growth_system_generator, positions, platf
         added_energy_components = compute_potential_components(mod_context)
```

Comment:
Instead of changing the `DEFAULT_PLATFORM` in tests/utils.py, we want to change this line to 

```
added_energy_components = compute_potential_components(mod_context, platform=platform)
```